demisto_sdk/commands/common/constants.py

```diff
@@ -134,7 +137,8 @@ class FileType(Enum):
     FileType.GENERIC_DEFINITION: 'Objects',
     FileType.GENERIC_MODULE: 'Modules',
     FileType.GENERIC_TYPE: 'Object Types',
-    FileType.GENERIC_FIELD: 'Object Fields'
+    FileType.GENERIC_FIELD: 'Object Fields',
+    FileType.LISTS: 'Lists'
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas) here and everywhere else
```suggestion
    FileType.LISTS: 'Lists',
```